### DISCUSSION TITLE
Welcome to cypress-file-upload Discussions!

### DISCUSSION THREAD

**Discussion by abramenal** [abramenal/cypress-file-upload#245] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by intermx-karthick** [comment]
npm install --save-dev cypress-file-upload  -> installing not working.

PS F:\MyRepo\widgetworld> npm install --save-dev cypress-file-upload
npm WARN @swimlane/ngx-dnd@8.2.0 requires a peer of @angular/common@^8.0.0 || ^9.0.0 || ^10.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @swimlane/ngx-dnd@8.2.0 requires a peer of @angular/core@^8.0.0 || ^9.0.0 || ^10.0.0 but none is installed. You must install peer 
dependencies yourself.
npm WARN less-loader@8.0.0 requires a peer of webpack@^5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ng-http-loader@8.0.0 requires a peer of @angular/common@^10.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ng-http-loader@8.0.0 requires a peer of @angular/core@^10.0.0 but none is installed. You must install peer dependencies yourself. 
npm WARN ng-mat-select-infinite-scroll@2.1.1 requires a peer of @angular/common@>=6.0.0 <11.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ng-mat-select-infinite-scroll@2.1.1 requires a peer of @angular/core@>=6.0.0 <11.0.0 but none is installed. You must install peer 
dependencies yourself.
npm WARN ng-mat-select-infinite-scroll@2.1.1 requires a peer of @angular/material@>=6.0.0 <11.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-widget-grid@10.0.1 requires a peer of @angular/common@^10.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-widget-grid@10.0.1 requires a peer of @angular/core@^10.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN cypress-file-upload@3.5.3 requires a peer of cypress@^3.0.0 but none is installed. You must install peer dependencies yourself.

npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@intermx%2fui-platform - Not found
npm ERR! 404
npm ERR! 404  '@intermx/ui-platform@1.3.14' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'widgetworld'
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\shyam\AppData\Roaming\npm-cache\_logs\2021-10-15T10_33_19_087Z-debug.log
PS F:\MyRepo\widgetworld>


Please Advice Me!!!

**Reply by abramenal** [reply]
I think it might be related to your own local setup – likely you're missing a token for your private package registry. Unrelated to this package
